test_scripts/ATF_ShutDown_IGNITION_OFF.lua

```diff
@@ -0,0 +1,71 @@
+-- Requirement summary:
+-- [Data Resumption]:OnExitAllApplications(IGNITION_OFF) in terms of resumption 
+--
+-- Description:
```

Comment:
@LitvinenkoIra,
To be more clear description and based on general template, you can list as example:
-- Requirement summary:
-- [Data Resumption]:OnExitAllApplications(IGNITION_OFF) in terms of resumption 
-- .
-- Description:
-- In case SDL receives OnExitAllApplications(IGNITION_OFF), SDL must clean up any resumption-related data 
-- obtained after OnExitAllApplications( SUSPEND). SDL must stop all its processes, notify HMI via OnSDLClose and shut down.
-- .
-- 1. Used preconditions
--  HMI is running
--  One App is registered and activated on HMI
-- .
-- 2. Performed steps
-- Perform ignition Off
-- HMI sends OnExitAllApplications(IGNITION_OFF)
-- .
-- Expected result:
-- 1. SDL sends to App OnAppInterfaceUnregistered
-- 2. SDL sends to HMI OnSDLClose and stops working